Source/iop/Iop_NamcoArcade.cpp

```diff
@@ -486,6 +578,26 @@ void CNamcoArcade::SetButtonState(unsigned int padNumber, PS2::CControllerInfo::
 
 void CNamcoArcade::SetAxisState(unsigned int padNumber, PS2::CControllerInfo::BUTTON button, uint8 axisValue, uint8* ram)
 {
+	switch(button)
+	{
+	case 0:
```

Comment:
Could you replace the numbers here with proper constants? (ie.: `PS2::CControllerInfo::BUTTON::ANALOG_LEFT_X` and others)